capplets/about-me/mate-about-me-fingerprint.h

```diff
@@ -19,9 +19,13 @@
 
 #include <gtk/gtk.h>
 
+#ifndef __MATE_ABOUT_ME_FINGERPRINT_H__
```

Comment:
I think it should also be on the first line of the code

Reply:
done